src/controllers/router-store/index.tsx

```diff
@@ -400,7 +400,7 @@ export const RouteResourceEnabledSubscriber = createSubscriber<
   Actions,
   boolean
 >(RouterStore, {
-  selector: state => Boolean(state.route && state.route.resources),
+  selector: state => Boolean(state.route && (state.route as any).resources),
```

Comment:
Should this be moved too? It can probably be recreated via `createRouterSelector`  (if we don't want to import router store)

Reply:
good point